features/dd-sdk-android-session-replay-compose/consumer-rules.pro

```diff
@@ -4,5 +4,22 @@
 -keepnames class androidx.compose.runtime.ComposerImpl$CompositionContextImpl
 -keepnames class androidx.compose.runtime.CompositionImpl
 -keepnames class androidx.compose.runtime.RecomposeScopeImpl
--keepnames class androidx.compose.ui.platform.WrappedComposition
+-keepnames class androidx.compose.runtime.Composition
+-keepnames class androidx.compose.ui.platform.ComposeView
 -keepnames class androidx.compose.material.DefaultButtonColors
+-keepclassmembers class androidx.compose.ui.platform.WrappedComposition {
```

Comment:
`keepnames` wasn't sufficient?

Reply:
I tested it, it is not enough, it seems `keepnames` will keep the classes and members names in the condition that they are actually used, otherwise they will be removed in shrinking process.

`keepclassmembers` seems fit more in our condition, they will be kept in any case.